backend/portal/api/portal_api.py

```diff
@@ -520,16 +519,10 @@
     if version.published_at:
         raise MethodNotAllowedException("Cannot delete a published Collection through API -- contact CXG Admins")
 
-    if isinstance(resource_id, CollectionVersionId):
-        try:
-            get_business_logic().delete_collection_version(resource_id)
-        except CollectionIsPublishedException:
-            raise ForbiddenHTTPException() from None
-    elif isinstance(resource_id, CollectionId):
-        try:
-            get_business_logic().tombstone_collection(resource_id)
-        except CollectionDeleteException() as e:
-            raise ServerErrorHTTPException(e.errors) from e
+    try:
```

Comment:
These changes and everything related could have been a separate PR to make reviewing easier.

Reply:
Very small change, though you're right in the sense that I should have labeled it as tangetially related